### PR TITLE
AdventOfCode day 5 Task 2. Complete the task

### PR DESCRIPTION
**Pull request by AnaSeregina**

### CODE REVIEW

File: AdventOfCode2021/day5/day5_1.py [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+            p_end: Point = Point(end_point[0], end_point[1])
+            l: Line = Line(p_start, p_end)
             if (l.horizontal() or l.vertical()):
                 lines.append(l)
     return lines
 
 
-def points_number_of_two_overlaps(lines: List[Line]) -> int:
-    point_overlaps_number: Dict[Tuple[int, int], int] = {}
+def point_overlaps(lines: List[Line]) -> Dict[Point, int]:
```
Comment (sazonovkirill) [inline AdventOfCode2021/day5/day5_1.py:23]: By convention, functions names are verbs, so `get_point_overlaps` could be a better name.

File: AdventOfCode2021/day5/day5_1.py [hunk 2]
```diff
@@ -1,37 +1,45 @@
...
+    point_overlaps_amount: Dict[Point, int] = {}
```
Comment (sazonovkirill) [inline AdventOfCode2021/day5/day5_1.py:24]: In cases you create an object at the start of a function and then return it in the end you can call it `result`.

**Review by sazonovkirill — APPROVED** [review]

File: AdventOfCode2021/day5/day5_1.py (hunk 1 above)
Comment (AnaSeregina) [reply AdventOfCode2021/day5/day5_1.py:23]: got it

File: AdventOfCode2021/day5/day5_1.py (hunk 2 above)
Comment (AnaSeregina) [reply AdventOfCode2021/day5/day5_1.py:24]: ok

**Pull request merged**